Assets/Project/Scripts/Modules/GamePlayScene/Bottle/DynamicBottleController.cs

```diff
@@ -124,6 +124,7 @@ public async UniTask MoveAsync(Vector3 targetPosition, CancellationToken outerTo
             _tokenSourceForMoveAsync.Cancel();
             // 新しいトークンを発行する
             _tokenSourceForMoveAsync = new CancellationTokenSource();
```

Comment:
```suggestion
            _tokenSourceForMoveAsync.Dispose();
            _tokenSourceForMoveAsync = new CancellationTokenSource();
```
https://github.com/LITO-apps/Treevel-client/pull/1121#discussion_r703101440
ここの議論にあるように、`new`する前に一回`Dispose`して欲しいですが入れてもらえますでしょうか

Reply:
c3f7f23